docs/authentication/social-connections/tiktok.mdx

```diff
@@ -1,47 +1,75 @@
+  1. Save the **Authorized redirect URI** somewhere secure. Keep the modal and page open.
 
-## Before you start
+  ### Create your app in TikTok
 
-- You need to create a Clerk Application in your [Clerk Dashboard](https://dashboard.clerk.com/). For more information, check out our [Set up your application guide](/docs/quickstarts/setup-clerk).
-- You need to have a TikTok Developer account. To create one, visit the [TikTok for developers](https://developers.tiktok.com/) page and click on Login ➜ Sign Up
+  > [!TIP]
+  > If you're creating an app as an organization rather than as an individual developer, you must first [create an organization](https://developers.tiktok.com/organizations). For guidance on which option to choose, refer to the [Working with organizations guide](https://developers.tiktok.com/doc/working-with-organizations/).
 
-## Configuring TikTok social connection
+  1. On a separate page, go to the [TikTok Developer Portal](https://developers.tiktok.com/) and sign in.
+  1. In the top-right, select [**Developer Portal**](https://developers.tiktok.com/apps) then choose **Manage apps**. You'll be redirected to the **Manage apps** page.
+  1. Select **Connect an app**. Complete the form then select **Confirm**. You'll be redirected to your TikTok app's **Production** page.
+  1. In the **Basic information** section, complete the form.
+  1. Under **Platforms**, select your platform then paste the **Authorized Redirect URI** you saved from the Clerk Dashboard.
+  1. Select the **Verify URL properties** under any of the URL fields to verify your app URL. A modal will open.
+  1. Select **Verify properties**. For Property Type, select either **Domain** to verify ownership of the enter website, including subdomains, or **URL prefix** to verify ownership of a specific part of the website (e.g., `example.com/shop/`).
+  1. Enter your website and select **Verify**.
+  1. Follow the instructions in the modal to verify your domain with your host provider. After entering the necessary information in your hosting provider's DNS settings, select **Verified**. Once your property is verified, select **Ok**.
+  1. Select **Submit for review**. Keep this page open.
 
-First, you need to create a new TikTok app. Go to the [TikTok for developers](https://developers.tiktok.com/) page, login with your developer account and create a new app by clicking on the **My Apps** ➜ **Connect a new app** and following the on screen wizard.
+  > [!NOTE]
+  > Your app needs to be reviewed by TikTok before the registration completes. This process may take a few days.
 
-![Creating a new TikTok app](/docs/images/authentication-providers/tiktok/9a5749a9bf3f282c54af2a2b539ba372e4706574-1489x964.png)
+  ### Set the Client ID and Client Secret in your Clerk Dashboard
 
-Add an icon and a name for your new project and hit **Start**.
+  1. In your TikTok app's **App details** page, select the icons next to the **Client key** and **Client secret** to reveal them.
```

Comment:
You called it the **Production** page on line 46
> You'll be redirected to your TikTok app's **Production** page.
Let's align these two - either Production page or App Details page

```suggestion
  1. In your TikTok app's **App details** page, select the icons next to the **Client key** and **Client secret** to reveal them. Copy these values.
```